### PR TITLE
[core, react, vue]: Enhancement - Add Sepolia testnet

### LINKED ISSUE #1598: [Feature]: Add Sepolia test net to the supported chains
### Is your request related to a problem?

_No response_

### Feature Description

I noticed that Sepolia test net is not in the list of supported chains by web3-onboard. Is this something you plan to add to the library? If yes, is it easy to implement? I could do it but I'm not familiar with the source code and I would like to know if it is feasible :)

Here's the list of supported chains (defined [here](https://github.com/blocknative/web3-onboard/blob/develop/packages/core/src/utils.ts#L114-L131)):

```sh
export const chainIdToLabel: Record<string, string> = {
  '0x1': 'Ethereum',
  '0x3': 'Ropsten',
  '0x4': 'Rinkeby',
  '0x5': 'Goerli',
  '0x2a': 'Kovan',
  '0x38': 'Binance',
  '0x89': 'Polygon',
  '0xfa': 'Fantom',
  '0xa': 'Optimism',
  '0x45': 'Optimism Kovan',
  '0xa86a': 'Avalanche',
  '0xa4ec': 'Celo',
  '0x14a33': 'Base Goerli',
  '0x64': 'Gnosis',
  '0x63564C40': 'Harmony One',
  '0xa4b1': 'Arbitrum'
}
```

### Alternative Solutions

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by Adamj1232**
### Description
Add Sepolia testnet to native support(chain icon and background color - both point to the Eth mainnet values)

Closes #1598 

### Checklist
- [x] Increment the version field in `package.json` of the package you have made changes in following [semantic versioning](https://semver.org/) and using alpha release tagging
- [x] Check the box that allows repo maintainers to update this PR
- [x] Test locally to make sure this feature/fix works
- [x] Run `yarn check-all` to confirm there are not any associated errors
- [x] Confirm this PR passes Circle CI checks
- [x] Add or update relevant information in the documentation

### CODE REVIEW

**Review by leightkt — APPROVED** [review]

File: packages/core/src/utils.ts [hunk 1]
```diff
@@ -103,6 +103,7 @@ export const toHexString = (val: number | string): string =>
 export function chainIdToHex(chains: (Chain | ChainWithDecimalId)[]): Chain[] {
   return chains.map(({ id, ...rest }) => {
     const hexId = toHexString(id)
+    console.log(hexId)
```
Comment (aaronbarnardsound) [inline packages/core/src/utils.ts:106]:
```suggestion

```

**Review by aaronbarnardsound — APPROVED** [review]
Just a sneaky log in there to remove.

**Comment by lewis-kori** [comment]
@Adamj1232 Thank you so much for the amazing work.
Just one more thing I noticed.
The notification callback in the [notify package](https://onboard.blocknative.com/docs/modules/core#initialization) doesn't receive events for the Sepolia test net. We use this to record payments, analytics among other things

I wasn't sure where to reach out with this.

**Comment by Adamj1232** [comment]
@lewis-kori We are very happy that you are enjoying web3-onboard!
We currently don't have transaction notifications running on Sepolia but it is on our roadmap and will be prepped as Goerli wind-down

**Pull request merged**